README.md

```diff
@@ -82,19 +82,27 @@ It downloads the latest version of ZeroNet then starts it automatically.
 
 #### Debian
 
-* `sudo apt-get update`
-* `sudo apt-get install msgpack-python python-gevent`
-* `wget https://github.com/HelloZeroNet/ZeroNet/archive/master.tar.gz`
-* `tar xvpfz master.tar.gz`
-* `cd ZeroNet-master`
-* Start with `python zeronet.py`
-* Open http://127.0.0.1:43110/ in your browser and enjoy! :)
+```sh
+sudo apt-get update
+sudo apt-get install msgpack-python python-gevent
+wget https://github.com/HelloZeroNet/ZeroNet/archive/master.tar.gz
+tar xvpfz master.tar.gz
+cd ZeroNet-master
+# to start:
+python zeronet.py
+xdg-open http://127.0.0.1:43110/
```

Comment:
I've just ran this, and realised that it stays running - i.e., you can't execute `xdg-open` because `python zeronet.py` stays running.